modules/mono/editor/GodotTools/GodotTools/Build/BuildOutputView.cs

```diff
@@ -120,13 +120,13 @@ private void LoadIssuesFromFile(string csvFile)
         private void IssueActivated(int idx)
         {
             if (idx < 0 || idx >= _issuesList.ItemCount)
-                throw new IndexOutOfRangeException("Item list index out of range");
+                throw new ArgumentOutOfRangeException(nameof(idx), "Item list index out of range.");
 
             // Get correct issue idx from issue list
             int issueIndex = (int)_issuesList.GetItemMetadata(idx);
 
             if (issueIndex < 0 || issueIndex >= _issues.Count)
-                throw new IndexOutOfRangeException("Issue index out of range");
+                throw new IndexOutOfRangeException("Issue index out of range.");
```

Comment:
Forgot to change to ArgOutOfRange?

Reply:
Or maybe InvalidOperation as I think this would not be the argument's fault but the object's state.